components/post_view/post_body/post_body.tsx

```diff
@@ -156,7 +168,12 @@ export default class PostBody extends React.PureComponent {
         }
 
         let fileAttachmentHolder = null;
-        if (((post.file_ids && post.file_ids.length > 0) || (post.filenames && post.filenames.length > 0)) && this.props.post.state !== Posts.POST_DELETED) {
+        if (
+            ((post && post.file_ids && post.file_ids.length > 0) ||
+        (post && post.filenames && post.filenames.length > 0)) &&
+      this.props.post &&
+      this.props.post.state !== Posts.POST_DELETED
```

Comment:
@Willyfrog - Somehow I  am not able to remove the `package-lock.json`. I tried few things but couldn't remove it from the PR. 
I tried these steps:

`git reset HEAD^ package-lock.json `
`git commit --amend --no-edit`
`git push -f`

Reply:
since you didn't add it the previous commit, that won't work.

you can probably check it out from your local master to fix it:
`git checkout master -- package-lock.json`

Reply:
I tried this... but seems like it didn't remove from the PR